doc/conf.py

```diff
@@ -58,8 +58,8 @@
 
 # General information about the project.
 project = u'DISCOS Simulators'
-copyright = u'2017, Marco Buttu, Giuseppe Carboni'
-author = u'Marco Buttu, Giuseppe Carboni'
+copyright = u'2017, Giuseppe Carboni, Marco Buttu'
```

Comment:
Keep the chronological order for people partecipating to the project, or alphabetical order (surname). If you want to specify that you are the mantainer:

```
copyright = u'2017, Marco Buttu, Giuseppe Carboni'
author = u'Marco Buttu, Giuseppe Carboni'
mantainer = u'Giuseppe Carboni'
```

Here an example: https://www.sphinx-doc.org/en/master/internals/authors.html